### ISSUE TITLE
Cannot load data from local (Failed: Internal Server Error (500))

### ISSUE THREAD

**Issue by iamjli** [fraenkel-lab/galaxy-neurolincs#3]
Tried uploading a few local files, but I'm getting error 500. 
![image](https://user-images.githubusercontent.com/7219365/35744762-2668e724-080f-11e8-97d1-4e5773492adf.png)

**Comment by iamjli** [comment]
FTP upload of the same file works. Probably a low priority bug.

**Comment by alexlenail** [comment]
Yes, currently HTTP uploads are broken on our galaxy for mysterious reasons
